### PR TITLE
feature: add title, description, date to create document

### PR DESCRIPTION
**Pull request by hsiliev**
Add title, description and date to create document flow
Adapt tests

Solves: PZ-4050

### CODE REVIEW

File: src/main/app/src/app/taken/taak-view/taak-view.component.ts [hunk 1]
```diff
@@ -297,6 +297,7 @@ export class TaakViewComponent
       "DA3A76D24DFD48C9837B03E47BC701FB";
     documentCreationData.smartDocumentsTemplateId =
       "7B7857BB9959470C82974037304E433D";
+    documentCreationData.title = "gemaakt door SmartDocuments";
```
Comment (bas-info-nl) [inline src/main/app/src/app/taken/taak-view/taak-view.component.ts:300]: Static strings should be declared as a const. And can then be re-used in other places.

File: src/main/java/net/atos/zac/util/time/ZonedDateTimeParamConverterProvider.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.time.ZonedDateTime;
+
+import jakarta.ws.rs.ext.ParamConverter;
+import jakarta.ws.rs.ext.ParamConverterProvider;
+import jakarta.ws.rs.ext.Provider;
+
+/**
+ * Solves missing valueOf() and string constructor for ZonedDateTime class
```
Comment (bas-info-nl) [inline src/main/java/net/atos/zac/util/time/ZonedDateTimeParamConverterProvider.java:12]:
```suggestion
 * Provider for ZonedDateTime jakarta web-service parameter converter.
 * @see ZonedDateTimeParamConverter
```

**Review by bas-info-nl — APPROVED** [review]
Minor comments only

File: src/main/kotlin/net/atos/zac/documentcreation/model/DocumentCreationDataAttended.kt [hunk 3]
```diff
@@ -5,13 +5,22 @@
 package net.atos.zac.documentcreation.model
 
 import net.atos.client.zgw.zrc.model.Zaak
+import java.time.ZonedDateTime
 
 data class DocumentCreationDataAttended(
-    val taskId: String? = null,
+    val zaak: Zaak,
+
+    val taskId: String?,
```
Comment (infonl-marcel) [inline src/main/kotlin/net/atos/zac/documentcreation/model/DocumentCreationDataAttended.kt:13]: I aassume this is the ID used for handshaking/verification and securing we just talked about; correct? If correct, then I don't understand the optional; if not correct, pls ignore this remark and close directly

**Review by infonl-marcel — APPROVED** [review]

File: src/main/kotlin/net/atos/zac/documentcreation/model/DocumentCreationDataAttended.kt (hunk 3 above)
Comment (hsiliev) [reply src/main/kotlin/net/atos/zac/documentcreation/model/DocumentCreationDataAttended.kt:13]: `taskId` is used when creating document for taak inside zaak. As the same object can be used for creating a document in zaak directly the `taskId` is optional.

**Pull request merged**